class.jetpack.php

```diff
@@ -6782,6 +6788,52 @@ public static function build_raw_urls( $url ) {
 		return $url;
 	}
 
+	/**
+	 * Builds an URL using the jetpack.com/redirect service
+	 *
+	 * @param string        $source The URL handler registered in the server
+	 * @param array|string  $args {
+	 *
+	 * 		Additional arguments to build the url
+	 *
+	 * 		@param string $site URL of the current site
+	 * 		@param string $path Additional path to be appended to the URL
+	 * 		@param string $query Query parameters to be added to the URL
+	 * 		@param string $anchor Anchor to be added to the URL
+
+	 * }
+	 *
+	 *
+	 * @return void
+	 */
+	public static function build_redirect_url( $source, $args = array() ) {
+
+		$url = 'https://jetpack.com/redirect';
+
+		$args = wp_parse_args($args);
+		$accepted = array( 'site', 'path', 'query', 'anchor' );
+
+		$add = array(
```

Comment:
what's `$add` means in this context? Can we rename it make it's meaning more explicit?

same goes to `$k` & `$arg`. from my understanding, `$k` is an argument/param, and `$arg` is the value of that param. 

UPD: I missed the "In Progress" label, so ignore me if it's not relevant :)

Reply:
yes, this is still a quick proof of concept to explore how to approach the problem. Not sure yet if we are going to go this way. But thanks for the feedback anyway!

Reply:
Fixed it in https://github.com/Automattic/jetpack/pull/15140/commits/ea9b02aeb4d34d210b3831b3584ceeacb8410464 anyway... I'm always the one picking on other to properly name variables.. hate to be caught on that!